Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1091,74 @@ namespace AtomSampleViewer
         }
 
     }
+
+    void ScriptReporter::ExportTestResults()
+    {
+        uint32_t totalAssertCount = 0;
+        uint32_t totalErrorCount = 0;
+        uint32_t totalWarningCount = 0;
+        uint32_t totalScreenshotErrorCount = 0;
+        uint32_t totalScreenshotWarningCount = 0;
+        
+        for (const ScriptReport& scriptReport : m_scriptReports)
+        {
+            totalAssertCount += scriptReport.m_assertCount;
+            totalErrorCount += scriptReport.m_generalErrorCount;
+            totalWarningCount += scriptReport.m_generalWarningCount;
+            totalScreenshotErrorCount += scriptReport.m_screenshotErrorCount;
+            totalScreenshotWarningCount += scriptReport.m_screenshotWarningCount;
+
+            AZStd::string assertLogLine = AZStd::string::format("Asserts: %i \n", totalAssertCount);
+            AZStd::string errorsLogLine = AZStd::string::format("Errors: %i \n", totalErrorCount);
+            AZStd::string warningsLogLine = AZStd::string::format("Warnings: %i \n", totalWarningCount);
+            AZStd::string screenshotErrorsLogLine = AZStd::string::format("Screenshot errors: %i \n", totalScreenshotErrorCount);
+            AZStd::string screenshotWarningsLogLine = AZStd::string::format("Screenshot warnings: %i \n", totalScreenshotWarningCount);
+            AZStd::string failedScreenshotsLogLine = "\nFailed screenshot test info below.\n";
```

Comment:
All of the %i should be %u because they are unsigned integers

nit- if you're not going to modify the variables after you create them you should prefix them with const but it's not going to hurt anything